Dizzy/Screens/PlaceStory/PlaceStoryVM.swift

```diff
@@ -38,26 +38,29 @@ protocol PlaceStoryVMType {
 
 final class PlaceStoryVM: PlaceStoryVMType {
     
-    let place: PlaceInfo
+    var place: PlaceInfo
     weak var delegate: PlaceStoryVMDelegate?
     weak var navigationDelegate: PlaceStoryVMNavigationDelegate?
     
     var displayedImageIndex = -1
     let delay = 1000.0
     var commentsInteractor: CommentsInteractorType
     var storiesInteractor: StoriesInteractorType
+    var placesIteractor: PlacesInteractorType
```

Comment:
`PlacesInteractor` need to be `let`,
it does no suppose to have delegate anymore (as i mentioned on the previous comments)
because right now after we set it's delegate to be `self` we cancel the delegation of `DiscoveryVM`
so everone who uses this interactor need to get the places through the bind and not the delegate

Reply:
Done